### PR TITLE
fix: describe proofreader role

### PR DESCRIPTION
**Pull request by dakeshi**
- add limited proofreader section
- add some missing url link

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Here are some guidelines to keep in mind as a translator:
 - If you find an error in the source English docs, open a pull request on the [electron/electron](https://github.com/electron/electron/tree/master/docs) repository.
 - If you've been working as a translator and want to have more influence over the approved translations in your language, let us know and we'll make you a proofreader.
 
-Electron's localization effort uses [Crowdin], an awesome platform for 
+Electron's localization effort uses [Crowdin](https://crowdin.com/project/electron), an awesome platform for
```
Comment (zeke) [inline contributing.md]: This `[Crowdin]` link already has a reference toward the bottom of the doc.

File: contributing.md [hunk 2]
```diff
@@ -42,14 +42,14 @@ more before applying. When contacting us, please include a link to your existing
...
-it syncs with Crowdin, and how to make technical changes.
+it syncs with Crowdin, how to make technical changes, and how to configure the proofreader role properly.
 
 **If you're just here to translate, you probably don't need to read this.**
 
 ### Environment Setup
 
 To fetch the latest docs, you need a GitHub token. Visit
-github.com/settings/tokens/new](https://github.com/settings/tokens/new)
+[https://github.com/settings/tokens/new](https://github.com/settings/tokens/new)
```
Comment (zeke) [inline contributing.md]: The protocol is not needed in the link text. it's cleaner without.

File: contributing.md [hunk 3]
```diff
@@ -124,3 +124,15 @@ content
...
+
+### Limited Proofreader Role
+
+There are two options to configure the proofreader role on crowdin.
+(You can change the role of translators in Translators tab on the project setting)
+
+- General proofreader: translate, approve strings for all languages.
+![general_proofreader](https://user-images.githubusercontent.com/8563047/38429142-8b8dc10e-39f8-11e8-9278-14314b557ca3.png)
+- The proofreader for languages: give a proofreader role for specific language
+![limited_proofreader](https://user-images.githubusercontent.com/8563047/38429156-96222196-39f8-11e8-9f1a-9ce012ecfa1d.png)
```
Comment (zeke) [inline contributing.md]: Can you show in the screenshot or explain in words how this languages sidebar is opened?

Comment (dakeshi) [reply contributing.md]: Good idea 👍  It would be better to attach gif.

File: contributing.md (hunk 2 above)
Comment (dakeshi) [reply contributing.md]: it makes sense.

**Comment by dakeshi** [comment]
Thank you for your review @zeke. fixed in 7a964af

**Pull request merged**